<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QItem</class>
 <widget class="QWidget" name="QItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>84</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QItem</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="checkFinished">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelPriority">
         <property name="text">
          <string>优先级</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelName">
         <property name="text">
          <string>name</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelete">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelDeadline">
       <property name="text">
        <string>无到期时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRepeat">
       <property name="text">
        <string>不重复</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRemind">
       <property name="text">
        <string>无提醒时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelGroupName">
       <property name="text">
        <string>我的待办</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
